### PR TITLE
disabled breadcrumb

### PR DESCRIPTION
**Pull request by ignasveleckisnc**
# What

- What was done in this Pull Request
- How was it before
- Screenshots
- Any other information that may be useful for the dev to review Pull Request

## Compatibility

- [x] Does this change maintain backward compatibility?

### CODE REVIEW

File: src/components/breadcrumbs/Breadcrumbs.tsx [hunk 1]
```diff
@@ -71,7 +79,9 @@ export function Breadcrumbs({ list, Component }: BreadcrumbsListProps) {
             {Component ? (
               <Component to={br.uri || ""}>{br.label}</Component>
             ) : (
-              <a href={br.uri || ""}>{br.label}</a>
+              <StyledLink isDisabled={br.isDisabled} href={br.uri || ""}>
```
Comment (mrMarshalX) [inline src/components/breadcrumbs/Breadcrumbs.tsx:82]: `br.uri || ""` is redundant (looking at the type. `Uri` can be only of `string` value.

File: nordcloud-gnui-8.14.0.tgz
Comment (nowyDEV) [inline nordcloud-gnui-8.14.0.tgz:1]: rm

File: src/components/breadcrumbs/Breadcrumbs.tsx [hunk 2]
```diff
@@ -62,6 +45,31 @@ const StyledBreadcrumbs = styled.nav`
...
+const StyledLink = styled.a<{ isDisabled?: boolean }>`
+  line-height: 1.125rem;
+  font-size: ${theme.fontSizes.sm};
+  font-weight: ${theme.fontWeights.regular};
+  font-family: ${theme.fonts.body};
+  text-decoration: none;
+  transition: ${theme.transition};
+  color: ${(props) =>
+    props.isDisabled ? theme.color.text.text02 : theme.color.interactive.link};
+  pointer-events: ${(props) => (props.isDisabled ? "none" : "auto")};
```
Comment (nowyDEV) [inline src/components/breadcrumbs/Breadcrumbs.tsx:57]: The styles for `disabled` state should be defined in a separate variable and conditionally applied all at once.
Mixing every style here and adding conditional statements for every line increases complexity and reduces readability.

File: src/components/breadcrumbs/Breadcrumbs.tsx [hunk 3]
```diff
@@ -62,20 +89,14 @@ const StyledBreadcrumbs = styled.nav`
...
-            ) : (
-              <a href={br.uri || ""}>{br.label}</a>
-            )}
-          </li>
-        ))}
-      </ul>
-    </StyledBreadcrumbs>
-  );
-}
+const disaledLinkStyles = css`
```
Comment (ErnestasZaloga) [inline src/components/breadcrumbs/Breadcrumbs.tsx:92]:
```suggestion
const disabledLinkStyles = css`
```

File: src/components/breadcrumbs/Breadcrumbs.tsx [hunk 4]
```diff
@@ -1,17 +1,58 @@
...
   list: BreadcrumbsList[];
-  Component?: React.FC<{ to: string }>;
+  Component?: React.FC<{ to: string; isDisabled?: boolean }>;
 };
 
+export function Breadcrumbs({ list, Component }: BreadcrumbsListProps) {
+  return (
+    <StyledBreadcrumbs>
+      <ul>
+        {list.map((br) => (
```
Comment (ErnestasZaloga) [inline src/components/breadcrumbs/Breadcrumbs.tsx:20]: Maybe rename `br` to `breadcrumb`?

**Review by ErnestasZaloga — APPROVED** [review]

**Review by mariusd-ibm — APPROVED** [review]

**Pull request merged**